TestEnvironment/demo-static-parent-issue/MyClass.php

```diff
@@ -10,6 +10,14 @@ public function load()
 		$refreshedEntity = self::refreshEntity($entity);
 		$refreshedEntity->entityMethod(); // auto-complete does not work for this
 	}
+
+	public function loadStaticProperty()
+	{
+		$user = StaticPropertyDataProvider::$user->getAccessToken();
+
+		$refreshedAccessToken = self::refreshEntity($user);
+		$refreshedAccessToken->isValid(); // auto-complete does not work for this
```

Comment:
@pbyrne84 JavaScript gets `\StaticPropertyDataProvider.user.getAccessToken` in this case